src/models.py

```diff
@@ -175,11 +175,36 @@ def upgrade_onethreeoh(mod):
     )
 
 
+def upgrade_onethreeoh_to_onethreeone(mod):
```

Comment:
Should have pointed this out earlier today -- this add-on uses [semantic versioning](https://semver.org) so this change will need to go out in a new feature release numbered 1.4.0. If you can change the wording/docstrings to match that would be great.